NextcloudTalk/BaseChatViewController.swift

```diff
@@ -1590,52 +1634,82 @@ import QuickLook
         audioFileName += ".mp3"
 
         let activeAccount = NCDatabaseManager.sharedInstance().activeAccount()
+        NCAPIController.sharedInstance().setupNCCommunication(for: activeAccount)
+        let chatFileController = NCChatFileController()
+        chatFileController.initDownloadDirectory(for: activeAccount)
+
+        let destinationFilePath = "\(chatFileController.tempDirectoryPath)/\(audioFileName)"
+
+        let temporaryMessage = self.createTemporaryMessage(
+            message: audioFileName,
+            replyTo: nil,
+            messageParameters: "\(destinationFilePath)",
+            silently: false,
+            isVoiceMessage: true
+        )
+
+        chatFileController.setFileStatusFrom(temporaryMessage.file())
+
+        _ = chatFileController.moveFileToTemporaryDirectory(
+            fromSourcePath: self.recorder!.url.path,
+            destinationPath: destinationFilePath
+        )
```

Comment:
If you don't care about the methods return value, why having a return value in the first place?
I guess if we can't move it, we should not show a temporary message?

Reply:
Fixed